lib/OpenQA/WebAPI.pm

```diff
@@ -501,6 +501,7 @@ sub startup ($self) {
       ->name('apiv1_put_parent_group_comment')->to('comment#update');
     $api_ra->delete('/parent_groups/<parent_group_id:num>/comments/<comment_id:num>')
       ->name('apiv1_delete_parent_group_comment')->to('comment#delete');
+    $api_ru->post('/comments')->name('apiv1_post_comments')->to('comment#create_many');
```

Comment:
I would be careful for two reasons:
1. We should consider complete CRUD
2. "comment" and "comments" are quite similar

Should we consider a different name?

Reply:
> We should consider complete CRUD

We have complete CRUD for comments. Of course I could add routes to also get, update and delete many comments at once by just specifying multiple comment IDs. However, that wouldn't be required for the ticket. I could nevertheless add those routes if you like. They'd only be usable via the CLI of course and one had to use some jq/bash magic to make e.g. delete previously created comments again in one go (based on the audit event¹).

Note that another missing part of this route is the possibility to specify group and parent group IDs. So far one can only create multiple comments for jobs. I also didn't implement group comments because it is not required for the ticket.

Note that with APIs we can always add new stuff but never remove old stuff. This speaks for only adding what we know we need.

> Should we consider a different name?

Maybe `apiv1_post_many_comments`? But I don't think the name we chose here really matters and I'd rather keep it short.

---

¹ I suppose I could also add a button in the audit log table that would actually make use of the delete route to undo a `openqa_comments_create` event.

Reply:
The latter makes sense I guess